src/devices/Ft4222/samples/Ft4222.sample.cs

```diff
@@ -45,13 +47,15 @@ static void Main(string[] args)
             if (key.KeyChar == '2')
                 TestSpi();
 
+            if (key.KeyChar == '3')
+                TestGpio();
         }
 
         private static void TestI2c()
         {
-            Ft4222I2c ftI2c = new Ft4222I2c(new I2cConnectionSettings(0, Bno055Sensor.DefaultI2cAddress));
+            var ftI2c = new Ft4222I2c(new I2cConnectionSettings(0, Bno055Sensor.DefaultI2cAddress));
 
-            Bno055Sensor bno055Sensor = new Bno055Sensor(ftI2c);
+            var bno055Sensor = new Bno055Sensor(ftI2c);
```

Comment:
nice test!

Reply:
Thanks :-) This is a complicated sensor, so if working with this one, lots of chances it will work with all the others ;-)